### ISSUE TITLE
Where is the Auxiliary BN in your codes?

### ISSUE THREAD

**Issue by yangjiao-tj** [tingxueronghua/pytorch-classification-advprop#5]
I have read the AdvResNet in your net.py,  but I can not find anything about the Auxiliary BN, which is an essential part in this paper you reproduced.

**Comment by tingxueronghua** [comment]
Sorry for replying late! I am busy with my undergraduate thesis project these days.
Yes, in net.py, we don't specify the normalization layer, and we send MixBatchNorm2d in imagenet.py into the model, which contains main and auxiliary batch normalization layers.
You could check ResNet in net.py, and could notice that the normalization layers are given by the variable "norm_layer", which could be specified outside. 
If you still have any other question, you could send emails to hanyc17@mails.tsinghua.edu.cn, which I should reply much faster.
